src/entities/AccessToken.ts

```diff
@@ -0,0 +1,33 @@
+import moment from "moment";
+
+export interface IAccessToken {
```

Comment:
```suggestion
export type AccessTokenData = {
```
Just an alternate suggestion for this. Might be better to just make this a type instead of an interface.

Reply:
Just updating this. Will push up shortly.